x/stake/keeper/post.go

```diff
@@ -0,0 +1,29 @@
+package keeper
+
+import (
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/rocket-protocol/stakebird/x/stake/types"
+)
+
+// Return post if one exists for (vendor_id | post_id)
+func (k Keeper) GetPost(ctx sdk.Context, vendorID uint32, postID uint64) (post types.Post, found bool) {
+	store := ctx.KVStore(k.storeKey)
+	key := types.PostKey(vendorID, postID)
+	value := store.Get(key)
+	if value == nil {
+		return post, false
+	}
+	types.ModuleCdc.MustUnmarshalBinaryLengthPrefixed(value, post)
+
+	return post, true
+}
+
+func (k Keeper) Post(ctx sdk.Context, postID uint64, vendorID uint32, body string, votingPeriod time.Duration) {
```

Comment:
What about `CreatePost` for creating  its more intuitive when I saw `Post` being called I thought it was a [getter](https://golang.org/doc/effective_go.html#Getters)

Reply:
Yeah totally agree, and I normally try to follow Effective Go guidelines. But Cosmos breaks this :). How about explicit `GetX()` and `CreateX()`? Or we can standardize on `X()` for getter and `CreateX()` for setter.

Reply:
Yeah Im a bit conflicted with this 😅  cosmos modules dont really follow those guidelines I think `GetX` and `CreateX()`  makes sense  to stick to cosmos like modules but  both are ok to me, I'll leave it to your preference  💪